javaslang-benchmark/src/test/java/javaslang/collection/ArrayBenchmark.java

```diff
@@ -77,43 +127,51 @@ public void tearDown() {
         }
 
         @Benchmark
-        public void java_mutable(Initialized state) {
+        public Object java_mutable(Initialized state) {
```

Comment:
Ok, I understand (not fully, because I don't know how dead code elimination works). Let's keep it as-is.